ibtk/include/ibtk/CartCellRobinPhysBdryOp.h

```diff
@@ -232,7 +232,7 @@ class CartCellRobinPhysBdryOp : public RobinPhysBdryPatchStrategy
      * \param ghost_width_to_fill  Integer vector describing maximum ghost width to fill over
      * all registered scratch components.
      */
-    void accumulateFromPhysicalBoundaryData(SAMRAI::hier::Patch<NDIM>& patch,
+    void accumulateFromPhysicalBoundaryData(const SAMRAI::hier::Patch<NDIM>& patch,
                                             double fill_time,
                                             const SAMRAI::hier::IntVector<NDIM>& ghost_width_to_fill) override;
```

Comment:
While making these things `const` is fine w.r.t. the type system, its a bit misleading because we modify the underlying `PatchData` object in a next step - could we leave these as-is?

Reply:
So do you think we should make _all_ of these `Patch` arguments non-`const` references? These functions really should _not_ be modifying these `Patch`es.

Reply:
I don't agree that it is misleading --- we are not changing the `Patch` object that is being passed in.